src/dmd/mars.d

```diff
@@ -753,6 +765,46 @@ private int tryMain(size_t argc, const(char)** argv, ref Param params)
     return status;
 }
 
+extern (C++) FileBuffer readFromStdin()
```

Comment:
Does anything in mars.d need to be extern(C++)? It's not as if there's anything shareable here with other compilers.

Reply:
Seems like private should be used here too.

Reply:
LDC is using a (rather heavily patched) `mars.d`. We don't need this particular function to be C++, as it will be called by the DMD main function too in our case; I was thinking GDC might need it.

Reply:
Nah, this is handled by gcc on my end.

Reply:
Nah, this is handled by gcc on my end.